jax/experimental/export/__init__.py

```diff
@@ -14,26 +14,59 @@
 # ==============================================================================
 
 from jax._src.export._export import (
-  minimum_supported_calling_convention_version,
-  maximum_supported_calling_convention_version,
-  Exported,
-  call_exported,  # TODO: deprecate
-  call,
-  DisabledSafetyCheck,
-  default_lowering_platform,  # TODO: deprecate
+    minimum_supported_calling_convention_version as minimum_supported_serialization_version,
+    maximum_supported_calling_convention_version as maximum_supported_serialization_version,
+    Exported,
+    DisabledSafetyCheck,
```

Comment:
Why are these not deprecated? Are we planning to keep `jax.experimental.export` around?

Also: shouldn't we also add deprecation warnings for `minimum_supported_serialization_version` and `maximum_supported_serialization_version`, which were renamed in #21828?

Reply:
I thought that we can only add deprecation warnings to functions. These are either integer values or Classes. I guess, for classes we could add the warning to the constructor. At the same time, I cannot imagine that somebody would use `jax.experimental.export` without using either `export()` or `deserialize()`. Those are the only ways to construct an `Exported`.

Reply:
We can add deprecation warnings to any exported symbol: it doesn't have to be callable.

I think we should do so here, otherwise any current users of these APIs who update JAX will get a hard `AttributeError` rather than a warning.

Reply:
Sounds good. Done!